[fix/ocis-sharing-names] Update sharing-related terms for oCIS

felix-schwarz opened the pull request:
## Description
- adds missing German translations (via synced Transifex)
- changes text in "Shared with me" when connected to an oCIS server
- adds icons for all sharing roles
- fully localizes Space "context menu" and Space Management UI

## Related Issue
https://github.com/owncloud/enterprise/issues/7320

## Screenshots (if appropriate):
### "Shared with me" connected to an oCIS server
Changed labels | Row action | Context menu action
--|--|--
<img width="282" height="531" alt="Bildschirmfoto 2025-09-30 um 22 31 50" src="https://github.com/user-attachments/assets/d03b2fdb-7db6-452d-8619-56ca78420c5b" /> | <img width="282" height="531" alt="Bildschirmfoto 2025-09-30 um 22 32 01" src="https://github.com/user-attachments/assets/50460d65-fb95-4deb-8700-c86b05814e9e" /> | <img width="282" height="531" alt="Bildschirmfoto 2025-09-30 um 22 31 57" src="https://github.com/user-attachments/assets/55fafb01-8a3a-4ce9-9657-7b248336e996" />

### "Create Link" and "Edit Link" views now localized in German
Create Link | Edit Link
---|---
<img width="282" height="531" alt="Bildschirmfoto 2025-10-01 um 10 06 23" src="https://github.com/user-attachments/assets/3be3b0ee-1b3e-478e-bf37-fb40082e27bf" /> | <img width="282" height="531" alt="Bildschirmfoto 2025-10-01 um 10 07 27" src="https://github.com/user-attachments/assets/2bc6036a-853f-40c8-b6b6-d32c03c0a3c2" />

### Icons for all sharing roles
(user name and avatar masked for privacy after taking the screenshots)

Invite for folder | Invite for file
--|--
<img width="564" height="1062" alt="Bildschirmfoto 2025-10-07 um 09 46 25" src="https://github.com/user-attachments/assets/ca69cd15-2701-44b4-b195-497d2af3c781" /> | <img width="564" height="1062" alt="Bildschirmfoto 2025-10-07 um 09 49 26" src="https://github.com/user-attachments/assets/5d146553-daa1-402e-a994-06b2b3cf1b93" />

### Spaces Menus fully localized
"Context Menu" | Spaces administration
--|--
<img width="564" height="1062" alt="Bildschirmfoto 2025-10-07 um 09 53 48" src="https://github.com/user-attachments/assets/1c89b98e-036a-4dd6-b260-9ac535ad3f97" /> | <img width="564" height="1062" alt="Bildschirmfoto 2025-10-07 um 10 35 26" src="https://github.com/user-attachments/assets/c6257708-f244-4700-b7f8-d3a49614fdcf" />

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

jesmrec commented:
About this PR:

- German translations to be accepted by german speakers. I will not approve/reject them.
- Icons in sharing: same, if the new icons are the expected ones, nothing to say from my side.

@dj4oC please check the screenshots above to assure they accomplish the customer expectation.

From QA, nothing else to say.

Feel free to move forward.

dj4oC commented:
> "Shared with me" connected to an oCIS server

OK

> "Create Link" and "Edit Link" views now localized in German

1. "Kann betrachten" bitte analog zum web-Client in "Kann anzeigen" ändern. 
2. unterhalb der Berechtigung "betrachten" zu "anzeigen" ändern. Dies tritt im rechten Screen auch weiter oben unterhalb von Link neben der Kopieren-Schaltfläche und ganz oben auf.

> "Create Link" and "Edit Link" views now localized in German

1. "Einladung" > "Freigaben", 
2. "Teilen mit" > "Mit Personen teilen", 
3. Button: "Hinzufügen" > "Setzen", 
4. Button "Einladung" > "Teilen"
5. Beschreibungen der Berechtigungen: "Ansehen" > "Anzeigen"

> Spaces Menus fully localized

1. Question: Are the role names ("Kann verwalten", "Kann bearbeiten", "Kann anzeigen") fetched from the Server?
2. Question: Is the "permanent link" hidden behind "Copy private link"? If so: Please rename to "Permanente Link"
3. Question: If the "private link" corresponds to the "permanent link", it would be more appropriate to isolate it from the public links. In the web client, it also appears in the detailed menu, not in the sharing menu.
4. Question: What is the difference between "Teilen" und "Mitglieder"? On Space Level there is only membership management
5. right Screen: 
5.1 Button "Einladung" > "Mitglieder hinzufügen" 
5.2 "Link erstellen" > "Link hinzufügen"; 
6. "Space deaktivieren" > "Deaktivieren"

felix-schwarz commented:
@dj4oC Danke für das Feedback.

> > "Create Link" and "Edit Link" views now localized in German
> 
> 1. "Kann betrachten" bitte analog zum web-Client in "Kann anzeigen" ändern.
> 2. unterhalb der Berechtigung "betrachten" zu "anzeigen" ändern. Dies tritt im rechten Screen auch weiter oben unterhalb von Link neben der Kopieren-Schaltfläche und ganz oben auf.

Erledigt.

Create Link | Edit Link
---|---
<img width="282" height="531" alt="Bildschirmfoto 2025-10-24 um 10 41 45" src="https://github.com/user-attachments/assets/75714d20-4d12-4644-9b70-9ac4c0f24579" /> | <img width="282" height="531" alt="Bildschirmfoto 2025-10-24 um 10 42 05" src="https://github.com/user-attachments/assets/e35200eb-e688-487f-b774-522aee710477" />

> > "Create Link" and "Edit Link" views now localized in German
> 
> 1. "Einladung" > "Freigaben",
> 2. "Teilen mit" > "Mit Personen teilen",
> 3. Button: "Hinzufügen" > "Setzen",
> 4. Button "Einladung" > "Teilen"

Erledigt.

Invite for folder | Invite for file
--|--
<img width="564" height="1062" alt="Bildschirmfoto 2025-10-24 um 10 57 18" src="https://github.com/user-attachments/assets/12d7d597-f663-4c27-8a67-03419d0114f9" /> | <img width="564" height="1062" alt="Bildschirmfoto 2025-10-24 um 10 56 49" src="https://github.com/user-attachments/assets/72d8debe-afa6-4ea2-98e5-611bbead0c62" />

> 5. Beschreibungen der Berechtigungen: "Ansehen" > "Anzeigen"

Diese Beschreibungen kommen vom Server und enthalten z.T. eine Dopplung: "**Ansehen**, herunterladen, bearbeiten und **anzeigen**…"

> > Spaces Menus fully localized
> 
> 1. Question: Are the role names ("Kann verwalten", "Kann bearbeiten", "Kann anzeigen") fetched from the Server?

Yes.

> 2. Question: Is the "permanent link" hidden behind "Copy private link"? If so: Please rename to "Permanente Link"

Done.

Sidenote on consistency: when entering this link in a browser, a message "Auflösen des privaten Links…" appears:

<img width="470" height="137" alt="Bildschirmfoto 2025-10-24 um 11 08 08" src="https://github.com/user-attachments/assets/5ea325b4-1518-4ec4-9522-8a056292157f" />

> 3. Question: If the "private link" corresponds to the "permanent link", it would be more appropriate to isolate it from the public links. In the web client, it also appears in the detailed menu, not in the sharing menu.

Done. I detached the entry into its own section:

<img width="564" height="1062" alt="Bildschirmfoto 2025-10-24 um 11 42 07" src="https://github.com/user-attachments/assets/34a91535-ccbf-4353-84b2-90a04d9da720" />

> 4. Question: What is the difference between "Teilen" und "Mitglieder"? On Space Level there is only membership management

This is indeed a duplication. Fixed that, too.

<img width="564" height="1062" alt="Bildschirmfoto 2025-10-24 um 11 44 37" src="https://github.com/user-attachments/assets/f77e2248-4ccf-4000-b932-73bdf1702af2" />

> 5. right Screen:
>    5.1 Button "Einladung" > "Mitglieder hinzufügen"
>    5.2 "Link erstellen" > "Link hinzufügen";

Fixed. (see second-last screenshot).

> 6. "Space deaktivieren" > "Deaktivieren"

Fixed. (see last screenshot)

felix-schwarz commented:
@dj4oC Please have a look at the new screenshots with the changes included and approve the PR in case everything's how it should be now.

dj4oC commented:
> > @dj4oC Please have a look at the new screenshots with the changes included and approve the PR in case everything's how it should be now.
> 
> This is almost perfect
> 
> unter: Invite for folder Invite for file steht "kann anzeigen --> "ansehen und herunterladen" --> korrekt
> 
> unter: Create Link Edit Link steht: "kann anzeigen --> "anzeigen und herunterladen" --> inkorrekt --> hier müsste auch "ansehen" stehen

If this is done, you are free to release
